changelog/unreleased/change-enable-live-reload-for-themes

```diff
@@ -0,0 +1,6 @@
+Change: Enable live reload for changes to themes
```

Comment:
Love it! Not a "Change" in terms of semver from my POV, though ;)

Reply:
True, changed it to `enhancement` now, thanks for the hint!

Reply:
Good point, agreed!